compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -2173,7 +2174,7 @@
                 },
                 {
                     "name": "content",
-                    "type": "TemplateMemberNode",
+                    "type": "Node",
```

Comment:
and this one too?

Reply:
ok, just saw the issue reported. +1 for this change. 
Shall we also remove the `TemplateMemberNode` from the syntax tree, since there are no more usages of it?

Reply:
Should I remove that in this PR as well?

Reply:
+1

Reply:
`XMLItemNode` extends from `TemplateMemberNode`. So should I change `XMLItemNode` to extend from the superclass of `TemplateMemberNode` instead? (which is `Node`)